sqlserver/datadog_checks/sqlserver/metrics.py

```diff
@@ -473,6 +473,82 @@ def fetch_metric(self, rows, columns):
             self.report_function(metric_name, column_val, tags=metric_tags)
 
 
+# https://docs.microsoft.com/en-us/sql/relational-databases/system-tables/backupset-transact-sql?view=sql-server-ver15
+class SqlDatabaseBackup(BaseSqlServerMetric):
+    CUSTOM_QUERIES_AVAILABLE = False
+    TABLE = 'msdb.dbo.backupset'
+    DEFAULT_METRIC_TYPE = 'gauge'
+    QUERY_BASE = (
+        "select count(backup_set_uuid) as backup_set_uuid_count, database_name "
+        "from {table} group by database_name".format(table=TABLE)
+    )
+
+    @classmethod
+    def fetch_all_values(cls, cursor, counters_list, logger):
+        return cls._fetch_generic_values(cursor, None, logger)
+
+    def fetch_metric(self, rows, columns):
+        database_name = columns.index("database_name")
+        value_column_index = columns.index(self.column)
+
+        for row in rows:
+            if row[database_name] != self.instance:
+                continue
+
+            column_val = row[value_column_index]
+            metric_tags = [
+                'database:{}'.format(str(self.instance)),
+            ]
+            metric_tags.extend(self.tags)
+            metric_name = '{}'.format(self.datadog_name)
+            self.report_function(metric_name, column_val, tags=metric_tags)
+
+
+class SqlDbFragmentation(BaseSqlServerMetric):
+    CUSTOM_QUERIES_AVAILABLE = False
+    TABLE = 'sys.dm_db_index_physical_stats'
+    DEFAULT_METRIC_TYPE = 'gauge'
+
+    QUERY_BASE = (
+        "select DB_NAME(database_id) as database_name, OBJECT_NAME(object_id) as object_name, "
+        "index_id, partition_number, fragment_count, avg_fragment_size_in_pages, "
+        "avg_fragmentation_in_percent "
+        "from {table} (null,null,null,null,null) "
+        "where fragment_count is not null".format(table=TABLE)
+    )
+
+    @classmethod
+    def fetch_all_values(cls, cursor, counters_list, logger):
+        return cls._fetch_generic_values(cursor, None, logger)
+
+    def fetch_metric(self, rows, columns):
+        value_column_index = columns.index(self.column)
+        database_name = columns.index("database_name")
+        object_name_index = columns.index("object_name")
+        index_id_index = columns.index("index_id")
+
+        for row in rows:
+            if row[database_name] != self.instance:
+                continue
+
+            column_val = row[value_column_index]
+            object_name = row[object_name_index]
+            index_id = row[index_id_index]
+
+            if self.cfg_instance.get('object_name') and self.cfg_instance.get('object_name') != object_name:
```

Comment:
```suggestion
            if self.cfg_instance.get('object_name') != object_name:
```
get returns None if its not set so `!=` will fail comparison. If you want to be more explicit you can do `if self.cfg_instance.get('object_name', '') != object_name`